### PR TITLE
Bump protobuf version to 3.6.1.2

### PR DESCRIPTION
**Pull request by davido**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -2,9 +2,17 @@ workspace(name = "io_bazel_rules_closure")
...
 load("//closure:repositories.bzl", "closure_repositories")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 closure_repositories()
 
+http_archive(
+    name = "bazel_skylib",
+    sha256 = "bbccf674aa441c266df9894182d80de104cabd19be98be002f6d478aaa31574d",
+    strip_prefix = "bazel-skylib-2169ae1c374aab4a09aa90e65efe1a3aad4e279b",
+    urls = ["https://github.com/bazelbuild/bazel-skylib/archive/2169ae1c374aab4a09aa90e65efe1a3aad4e279b.tar.gz"],
```
Comment (laurentlb) [inline WORKSPACE]: When you use a commit hash, please add the commit date in a comment.

Comment (davido) [reply WORKSPACE]: Resolved. I have updated to released version 0.6.0.

**Comment by davido** [comment]
It worth noting, that Gerrit Code Review cannot be built any more with upcoming 0.20 release.

**Comment by laurentlb** [comment]
@dslomov

**Comment by laurentlb** [comment]
@dslomov 
@philwo 
@buchgr 

Can someone please merge this?
(We really need more maintainers for this repo)

File: WORKSPACE [hunk 2]
```diff
@@ -2,6 +2,7 @@ workspace(name = "io_bazel_rules_closure")
 
 load("//closure/private:java_import_external.bzl", "java_import_external")
 load("//closure:repositories.bzl", "closure_repositories")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```
Comment (gkdn) [inline WORKSPACE]: Seems unused. Please revert.

File: closure/repositories.bzl [hunk 3]
```diff
@@ -675,26 +675,23 @@ def com_google_jsinterop_annotations():
...
         ],
     )
 
 def com_google_protobuf_js():
     http_archive(
         name = "com_google_protobuf_js",
         urls = [
-            "https://mirror.bazel.build/github.com/google/protobuf/archive/v3.5.1.tar.gz",
-            "https://github.com/google/protobuf/archive/v3.5.1.tar.gz",
+            "https://github.com/google/protobuf/archive/7b28271a61a3da0a37f6fda399b0c4c86464e5b3.tar.gz",
```
Comment (gkdn) [inline closure/repositories.bzl]: Currently all of the external deps here are mirrored from mirror.bazel.build. I'm not sure if that's a policy/best practice or not but your change doesn't follow that.

Comment (davido) [reply closure/repositories.bzl]: Can you (or someone else) upload https://github.com/protocolbuffers/protobuf/archive/v3.6.1.2.tar.gz" to https://mirror.bazel.build/github.com/google/protobuf/archive/v3.6.1.2.tar.gz? Then I can re-add new mirror URL.

Comment (gkdn) [reply closure/repositories.bzl]: I only have write access to rules_closure and not even sure if this is important.
Somebody from Bazel team should clarify if bazel mirrors has any value-add here.
@dslomov @laurentlb

**Comment by davido** [comment]
Doesn't seem to be a viable option, [because](https://github.com/bazelbuild/bazel/issues/6852) of this complication.

**Comment by davido** [comment]
As confirmed by @meteorcloudy, 3.6.1.2 is binary compatible with 3.6.1, even though, iit's totally unclear, what protobuf version is currently used by rules_closure.

File: closure/repositories.bzl (hunk 3 above)
Comment (laurentlb) [reply closure/repositories.bzl]: `should clarify if bazel mirrors has any value-add here`
I have no idea about the mirrors, but most other rules don't use them.

Comment (meteorcloudy) [reply closure/repositories.bzl]: TensorFlow uses bazel mirrors a lot, because it's more stable than the github link.

**Comment by laurentlb** [comment]
LGTM

cc @gkdn @nfelt @pcj

**Comment by meteorcloudy** [comment]
Thank @buchgr for merging this!
@davido Can you update rules_closure version in protobuf?

**Comment by davido** [comment]
@meteorcloudy @buchgr Thanks! Working on updating `rules_closure` in Gerrit.

**Pull request merged**